main.go

```diff
@@ -12,6 +12,8 @@ import (
 
 var (
 	verbose bool
+	//GitCommit git commit SHA
+	GitCommit string
 )
 
 func main() {
```

Comment:
Hi @viveksyngh 

Thank you for this PR!

Do we need to use this `GitCommit` in the version command or in the `main()`?

What I meant was to add SHASUM of the binary when run CI/CD and generate a release, but I think this is a great addition too and I would also like to merge this 👍 

Alex

Reply:
We can print this in main while running in verbose mode.